allauth/socialaccount/providers/google/tests.py

```diff
@@ -204,3 +204,15 @@ def test_social_email_verification_optional(self):
         self.assertEqual(len(mail.outbox), 1)
         self.login(self.get_mocked_response(verified_email=False))
         self.assertEqual(len(mail.outbox), 1)
+
+    def test_callback_android(self):
```

Comment:
I think this test is fine. The only functionality you add is blanking the callback_url, so as far as I'm concerned that's all you need to test. Provided, of course, that you tested this and it Acutally Works (tm)